src/ripple/protocol/impl/STTx.cpp

```diff
@@ -335,9 +351,9 @@ STTx::checkMultiSign () const
     auto const txnAccountID = getAccountID (sfAccount);
 
     // Determine whether signatures must be full canonical.
-    ECDSA const fullyCanonical = (getFlags() & tfFullyCanonicalSig)
-        ? ECDSA::strict
-        : ECDSA::not_strict;
+    bool const fullyCanonical = (getFlags() & tfFullyCanonicalSig)
+        ? true
+        : false;
```

Comment:
No need for the trinary `? true : false` here, either, especially since the variable is explicitly `bool`.